string class_id
uint32 class_type
float64 conf
uint16 x_bottom
uint16 y_bottom
uint16 x_top
uint16 y_top
